numerrin_wrapper/numerrin_mesh.py

```diff
@@ -0,0 +1,448 @@
+                pass
+            return face
+        except KeyError:
+            error_str = "Trying to get an non-existing edge with uuid: {}"
+            raise ValueError(error_str.format(uuid))
+
+    def get_cell(self, uuid):
+        """Returns a cell with a given uuid.
+
+        Returns the cell stored in the mesh
+        identified by uuid . If such cell do not
+        exists an exception is raised.
+
+        Parameters
+        ----------
+        uuid
+            uuid of the desired cell.
+
+        Returns
+        -------
+        Cell
+            Cell with id identified by uuid
+
+        Raises
+        ------
+        Exception
+            If the cell identified by uuid was not found
+
+        """
+
+        try:
+            pointLabels = numerrin.getelement(self.pool.ph, self.name,
+                                              3, self._uuidToNumLabel[uuid],
+                                              0)
+            puids = [self._numPointLabelToUuid[lbl] for lbl in pointLabels]
+            cell = Cell(puids, uuid)
+            return cell
+
+        except KeyError:
+            error_str = "Trying to get an non-existing cell with uuid: {}"
+            raise ValueError(error_str.format(uuid))
+
+    def add_point(self, point):
+        message = 'Point addition not supported yet'
+        raise NotImplementedError(message)
+
+    def add_edge(self, edge):
+        message = 'Edge addition not supported yet'
+        raise NotImplementedError(message)
+
+    def add_face(self, face):
+        message = 'Face addition not supported yet'
+        raise NotImplementedError(message)
+
+    def add_cell(self, cell):
+        message = 'Cell addition not supported yet'
+        raise NotImplementedError(message)
+
+    def update_point(self, point):
+        """ Updates the information of a point.
+
+        Gets the mesh point identified by the same
+        uuid as the provided point and updates its data
+        with the one provided with the new point.
+
+        Parameters
+        ----------
+        point : Point
+            Point to be updated
+
+        Raises
+        ------
+        KeyError
+            If the point was not found in the mesh
+
+        """
+
+        if point.uid not in self._uuidToNumLabel:
+            error_str = "Trying to update a non-existing point with uuid: "\
+                + str(point.uid)
+            raise KeyError(error_str)
+
+        for dkey in numvariables:
+            dataName = numname[dkey]
+            vname = self.name + dataName
+            if dkey in point.data:
+                try:
+                    vdata = list(self.pool.get_variable(vname))
+                    vdata[self._uuidToNumLabel[point.uid]] = point.data[dkey]
+                    self.pool.modify_variable(vname, tuple(vdata))
+                except:
+                    # create variable
+                    var = point.data[dkey]
+                    if type(var) is tuple:
```

Comment:
I mean, it is not clear why all tuple variables are interpreted as 3D vectors and non-tuples as scalars? That is, what happens if the ``var`` is a (3D) matrix of tuple type?

In other words, is there not a better or more self-explanatory way to decide what kind of variables (mainly rank if dimension always 3D) should be created into the pool?